src/webgpu/api/validation/resource_usages/textureUsageInPassEncoder.spec.ts

```diff
@@ -184,6 +185,10 @@ class TextureUsageTracking extends ValidationTest {
       (pass as GPURenderPassEncoder).draw(3, 1, 0, 0);
     }
   }
+
+  isWritableBinding(bindingType: GPUBindingType | 'render-target'): boolean {
```

Comment:
@kainino0x any place where we can put such helpers?

Reply:
It should be a new field in capability_info. In this case, a column in kTextureBindingTypeInfo.

Reply:
Hmm, we can add a new field in kTextureBindingTypeInfo. But the new field will be set true for only one single binding type and it might be used in only one single test file (this one only?). So adding a new field doesn't make too much sense compared to a helper function here or a simple statement like `bindingType === "writeonly-storage-texture"`. In addition, "render-target" is a resource usage but it is not a binding type, so statement like `kTextureBindingTypeInfo[bindingType].writable` will fail when we run `grunt pre`. For these reasons, maybe we should keep the code as what it is right now? @kainino0x , @Kangz .

Reply:
Having the writable set to true for a single binding type is not a problem, especially if we consider that in the future we'll have `storage-texture` that's also writable. However I'm not sure what to do for `render-target`. Maybe we could have a sum type that contains the binding type | render-target? I don't know enough Typescript to be sure.

Reply:
Any thoughts about this? @kainino0x . If we don't have a good solution for 'render-target', maybe we should proceed and merge this pr? I think the remain concern is really trivial: one line code like "bindingType === 'writeonly-storage-texture'" or a function like isWritableBinding is not hard to understand, and it doesn't hurt the readability a lot, per my understanding.

Reply:
If I'm understanding correctly, this function is no longer needed as explained below

Reply:
Sounds like I don't need to do any follow-ups on this pr and it can be merged?

Reply:
"IsWritableBinding" is misleading. The logic should just be inlined like it was before.

It could be replaced with a  function saying "bindingsConflict(a, b)" but we don't want to write that fully general function right now so we shouldn't.

Reply:
Done.